backends/tex_inspect_opengl.cpp

```diff
@@ -387,7 +387,7 @@ const GLchar *fragment_shader_glsl_120 = "#ifdef GL_ES\n"
                                          "        uv = Frag_UV;\n"
                                          "    vec2 texelEdge = step(mod(texel,vec2(1.0,1.0)),GridWidth);\n"
                                          "    float isGrid = max(texelEdge.x, texelEdge.y);\n"
-                                         "    vec4 ct = ColorTransform * texture2D(Texture, uv) + ColorOffset;\n"
+                                         "    vec4 ct = ColorTransform * texture(Texture, uv) + ColorOffset;\n"
```

Comment:
There is no texture function in GLSL 1.20, we need to stick with texture2D here.